.github/workflows/ci.yml

```diff
@@ -133,31 +137,75 @@ jobs:
         sudo ./ci/setup_ci_environment.sh
         sudo ./ci/install_gcc48.sh
     - name: setup cmake
-      run: |
-        sudo CC=/usr/bin/gcc-4.8 CXX=/usr/bin/g++-4.8 ./ci/setup_cmake.sh
+      env:
+        CC: /usr/bin/gcc-4.8
+        CXX: /usr/bin/g++-4.8
+        GOOGLETEST_VERSION: "1.10.0"
+      run: sudo -E ./ci/setup_cmake.sh
```

Comment:
nit: put setup_cmake and setup_grpc in to step as both need the same env?

Reply:
Thanks, `setup_cmake.sh` and `setup_grpc.sh` belong to the same step now.